test/factories/timer_session.rb

```diff
@@ -10,19 +10,22 @@
 
     trait :with_issues do
       after(:create) do |timer_session|
-        TimerSessionIssue.create!(
-          timer_session: timer_session,
-          issue: Issue.first
-        )
+        TimerSessionIssue.create!(timer_session:, issue: Issue.first)
       end
     end
 
     trait :with_time_entries do
       after(:create) do |timer_session|
-        TimerSessionTimeEntry.create!(
-          timer_session: timer_session,
-          time_entry: TimeEntry.first
-        )
+        TimerSessionTimeEntry.create!(timer_session:, time_entry: TimeEntry.first)
```

Comment:
I converted these traits into one-liners because rubocop complained about the line count